src/Behat/Symfony2Extension/ServiceContainer/Symfony2Extension.php

```diff
@@ -80,6 +80,12 @@ public function configure(ArrayNodeDefinition $builder)
                             ->end()
                             ->defaultTrue()
                         ->end()
+                        ->booleanNode('shared')
```

Comment:
`shared_container` ?

Reply:
Technically, it's the `kernel` that's being shared, which has a reference to the container.